### ISSUE TITLE
Add FRCP citation parsing

### ISSUE THREAD

**Issue by mlissner** [anseljh/casebot#11]
Can we do this too, so that when somebody says something like:

> sounds like a FRCP 5.2(a)(2) violation to me

It replies with:

> https://www.law.cornell.edu/rules/frcp/rule_5.2

Or something along those lines? I don't know if it's just me, but I don't have any of these things memorized. Maybe it'd be annoying for some people, but I'd adore this.

Bonus points if it can quote the rule somehow, but that's probably impossible.

**Comment by mlissner** [comment]
Hm, well, just providing the link provides the open graph info:

### Rule 5.2. Privacy Protection For Filings Made with the Court
> (a) Redacted Filings. Unless the court orders otherwise, in an electronic or paper filing with the court that contains an individual's social-security number, taxpayer-identification number, or birth date, the name of an individual known to be a minor, or a financial-account number, a party or nonparty making the filing may include only: (1) the last four digits of the social-security number and taxpayer-identification number;

That's pretty warm.

**Comment by anseljh** [comment]
Good idea! To clarify: do you mean it would watch messages in a channel it's invited to, and then respond to messages that match certain patterns?

**Comment by mlissner** [comment]
Yep, exactly.
